dvc/commands/commit.py

```diff
@@ -42,13 +44,6 @@ def add_parser(subparsers, parent_parser):
         help=COMMIT_HELP,
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
-    commit_parser.add_argument(
-        "-f",
-        "--force",
-        action="store_true",
-        default=False,
-        help="Commit even if hash value for dependencies/outputs changed.",
-    )
```

Comment:
In a way, removing prompts could be seen as a breaking change.

Reply:
Removing the option itself is more breaking, right?

Reply:
Also seems like a completely unrelated change

Reply:
Good point @daavoo. I will bring the flag back. But still the default behavior that it no longer fails is a breaking change.